### ISSUE TITLE
With the site down, is there a way to manually sync clipto data from desktop to phone?

### ISSUE THREAD

**Issue by paul1149** [clipto-pro/Desktop#159]
I just came across clipto last week, great prog, and now it's down. I had started copying snippets to the web interface, and planned to then sync them to my Android.

So now I've installed the AppImage on my Linux desktop, but I don't see where it stores its data.

Is this possible? Thanks.

**Comment by febryanvaldo** [comment]
How is it now? What app to replace this?

**Comment by paul1149** [comment]
I haven't been syncing.

**Comment by febryanvaldo** [comment]
Hmmm

**Comment by herrmann0319** [comment]
Xclipper is similar but not as feature rich. Should I cancel my subscription? What a shame.

**Comment by febryanvaldo** [comment]
> Xclipper is similar but not as feature rich. Should I cancel my subscription? What a shame.

Yeah, the only app is XClipper, i was on their premium plan (one time fee) long time ago, but the desktop app made my PC stutters. That's why I don't use it again.

**Comment by herrmann0319** [comment]
I think there are others. I remember trying a bunch a few years ago. These were just my two favorite. Like I said on the other thread I haven't tried the Xclipper PC app yet. I'm gonna give it a go this week or maybe even today. 

Makes your PC stutter? Something isn't right about that. Maybe it's conflicting with another program? I'll let you know what my experience is.

I guess I'm gonna have to cancel my Clipto subscription then since it's now quite useless. I wonder if his bank account deficit will make him remember he made this app again, haha.

**Comment by febryanvaldo** [comment]
Yeah, i think there are others. Like https://magiccopy.app for paid ones. It has web access, maybe i will try it. The second one would be JOIN by Joaoapps or The premium, EasyJoin (one time fee).

Yeah, the PC app (XClipper) made my PC stutters, i knew it, because i have 144Hz monitor, so when i typed (MonkeyType), i felt the stuttering, and when i close quit the XClipper program, the stutters was non existent.

**Comment by paul1149** [comment]
If you want to share the current clipboard entry in real time, you can do that *over wifi* with K-Connect. I would suggest grabbing the apk for clipto at this point.

**Comment by herrmann0319** [comment]
So I just tried setting up the Xclipper PC app with the complicated firebase instructional video and documentation which are both outdated but I figured it out. Then I notice that the premium plan only allows you to save upto 150 entries. What a waste!! All of that for nothing. I want something that will save an unlimited amount or at least 500 or 1000. I think I'm paying .99 cents a month for something like this with Clipto. 

I currently share the clipboard entry between phone and computer using the "Phone Link" application on windows. This allows me to copy and paste from phone to computer seamlessly. However, it does not save the things I copy or keep a log of of my clipboard like Clipto and Xclipper. Now that I'm checking it looks like Xclipper is only storing the last "TWO WEEKS" worth.

Just a tip for you guys. I also use this app called "Type Keeper" it logs every single thing I type from every app. So if I ever lose anything I can retrieve it again. Now that I think about it it ALSO saves your clipboard history as well. The interface for clips isn't as nice as the other two apps and no way to sync with computer.
 
Since the app is so useful I pay for a subscription for version 2.0 but you can also get a modded version of 1.0 that does the same thing minus clipboard. This app has saved my ass more times than I can count and it's saving every single thing I type. There is also security so you have to swipe to access it. 

Sure it's a bit shady having a keylogger on your phone but I figured I'd share this since its along the same lines and I find it useful. I'm gonna take a look at that magiccopy suggestion as well above. Thanks!

**Comment by herrmann0319** [comment]
> Yeah, i think there are others. Like https://magiccopy.app for paid ones. It has web access, maybe i will try it. The second one would be JOIN by Joaoapps or The premium, EasyJoin (one time fee).
> 
> Yeah, the PC app (XClipper) made my PC stutters, i knew it, because i have 144Hz monitor, so when i typed (MonkeyType), i felt the stuttering, and when i close quit the XClipper program, the stutters was non existent.

I'm gonna check these out and yea that is so weird with the stutter. Thanks for the suggestions. I'll report back if I come up with anything. 

Edit - MagicCopy has a 3.1 in the app store. Nothing but complaints and hasn't been updated in over a year. 

Edit - EasyJoin is an extremely cool app but I'm able to do all of these things with the "Link to Windows" app I'm using now. Share files, use apps, share clipboard, and even chat over SMS from my computer. It's free. 

It doesn't log your clipboard though which is what Clipto does and what we are after. All useful stuff though. Maybe it can come in handy for you guys I've been using it for years. Happy to share!

**Comment by febryanvaldo** [comment]
> I'm gonna check these out and yea that is so weird with the stutter. Thanks for the suggestions. I'll report back if I come up with anything.

Yeah, the stutters irritates me... That's why i don't use it again.

> Edit - MagicCopy has a 3.1 in the app store. Nothing but complaints and hasn't been updated in over a year.

Hmmm, maybe they don't need to update as often, as it already mature feature-wise? Maybe?

> Edit - EasyJoin is an extremely cool app but I'm able to do all of these things with the "Link to Windows" app I'm using now. Share files, use apps, share clipboard, and even chat over SMS from my computer. It's free.

Yeah, right now, i' using Phone Link aka Link to Windows, it's a very good app, and it's free. 

Actually i just need Universal Copy and Paste between devices though, and i don't need Clipboard logs, as i'm using Samsung Ecosystem right now, i think it's already solved. I can Sign In with the same Samsung Account, the same WiFi network, and the Clipboard just sync with each others. For Windows, i use Phone Link aka Link to Windows.

But it's always nice to have some option, like JOIN, EasyJoin, and i heard KDE Connect (https://kdeconnect.kde.org) also has this kind of feature (Universal Copy and Paste clipboard between devices). Might worth to check it in the future.

**Comment by Mike15041991** [comment]
Does anyone know how to save the local clipto database and transfer it to another device?

**Comment by herrmann0319** [comment]
> Actually i just need Universal Copy and Paste between devices though, and i don't need Clipboard logs...
> 
Speaking of Universal Copy I used to use it all the time but I found there were some things it wasn't working with. There's another app just like it called "TextCopy." I have both in my quick panel. They both compliment each other. One will copy text the other won't. Get the modded versions of course.  

However, since I got my S24 Ultra it has this feature built in it so I haven't been using these apps. You just hold your finger on the bar thats at the bottom of the screen and you can copy any text on the screen along with other "AI" Google Lens type stuff.

BTW Clipto is off the effing Play Store now...
